components/unordered-list/src/_mixin.scss

```diff
@@ -9,7 +9,9 @@
   box-sizing: border-box;
   font-family: var(--utrecht-document-font-family, inherit);
   font-size: var(--utrecht-unordered-list-font-size, var(--utrecht-document-font-size, inherit));
+  forced-color-adjust: none;
```

Comment:
Dit maakt het juist kapot in forced colors mode 🤔

Normaal:

<img width="1032" alt="Screenshot 2024-08-22 at 10 43 47" src="https://github.com/user-attachments/assets/780e382a-369c-4176-a9ab-55cea30eb498">

Forced colors:

<img width="1025" alt="Screenshot 2024-08-22 at 10 43 38" src="https://github.com/user-attachments/assets/f7472612-c872-4f5b-8795-c2ed3fddca60">

Heb je het wel getest met forced colors?

Reply:
Ik heb het probleem opgelost!